data/libraries.js

```diff
@@ -1,127 +1,160 @@
-        await libraryCollection.updateOne(
-            {_id: new ObjectId(libraryId)},
-            {$push: {comments: newComment}}
-        );
-    },
-    // getNumLikes
-    // getNumFavorites
-    // addLiketoComment
-    // addLiketoLibrary
+    const libraryCollection = await libraries();
+    await libraryCollection.updateOne(
+      { _id: new ObjectId(libraryId) },
+      { $push: { comments: newComment } }
+    );
+  },
+  // getNumLikes
+  // getNumFavorites
+  // addLiketoComment
+  // addLiketoLibrary
 
-    /** This function gets an array of Libaries by ownerID*/
-    async getLibrariesByOwnerID(ownerId) {
-        ownerId = validation.checkId(ownerId);
-        const librariesCollection = await libraries();
-        return await librariesCollection.find({ownerID: ownerId}).toArray();
-    },
-    /** This function will remove a library if the userId is equal to the ownerID*/
-    async removeLibrary(libraryId, userId) {
-        libraryId = validation.checkId(libraryId);
-        userId = validation.checkId(userId);
-        const libary = getLibraryById(libraryId)
-        if(userId ===libary.ownerID){
-        const librariesCollection = await libraries();
-        const deletionInfo = await librariesCollection.findOneAndDelete({
-            _id: ObjectId(libraryId)
-        });
-        if (deletionInfo.lastErrorObject.n === 0)
-            throw `Error: Could not delete post with id of ${libraryId}`;
-        return {...deletionInfo.value, deleted: true};
-        }
-        else{ 
-        throw `Error: Could not delete post with id of ${libraryId}`;
-        }
+  /** This function gets an array of Libaries by ownerID*/
+  async getLibrariesByOwnerID(ownerId) {
+    ownerId = validation.checkId(ownerId);
+    const librariesCollection = await libraries();
+    return await librariesCollection.find({ ownerID: ownerId }).toArray();
+  },
+  /** This function will remove a library if the userId is equal to the ownerID*/
+  async removeLibrary(libraryId, userId) {
+    libraryId = validation.checkId(libraryId);
+    userId = validation.checkId(userId);
+    const libary = getLibraryById(libraryId); // This is misspelt. Did you mean this?
+    if (userId === libary.ownerID) {
+      const librariesCollection = await libraries();
+      const deletionInfo = await librariesCollection.findOneAndDelete({
+        _id: ObjectId(libraryId),
+      });
+      if (deletionInfo.lastErrorObject.n === 0)
+        throw `Error: Could not delete post with id of ${libraryId}`; // Not sure if you meant to keep the word post here
+      return { ...deletionInfo.value, deleted: true };
+    } else {
+      throw `Error: Could not delete post with id of ${libraryId}`;
+    }
+  },
+  /**
+   * jcarr2
+   * @name formUpdate
+   * @param {String} libraryId - The ID of the library to change
+   * @param {Number} fullnessRating
+   * @param {Array<String>} genres
+   * @returns {Object} The updated object of the library
+   */
```

Comment:
Awesome jsdocs @JulienCarr2 

One small tip though, for the return value instead of type Object, write a custom object that has all the fields that updated library object will have